manifests/init.pp

```diff
@@ -1,52 +1,87 @@
-# Manages the enabling and disabling of FIPS
+# This module manages the enabling and disabling of FIPS on a system
+# It will set the kernel boot parametes and install/remove the dracut packages
+# and rebuild initramfs images.
+#
+# Changing the FIPS status of a system changes the cryptographic modules used.
+# This can affect existing keys and certificates and make them unusable.  Make
+# sure these affect are understood before changing the status.
+#
+# @param enabled
+#   If FIPS should be enable or disabled on the system.
+#
+# @param aesni
+#   NOTE: This parameter is controlled by params.pp
+#   This parameter indicates wether or not the system uses the
+#   Advanced Encryption Standard New Instructions set.
 #
 class fips (
-  Boolean $enabled = $::fips::params::enabled,
+  Boolean $enabled = simplib::lookup('simp_options::fips', { 'default_value' => true }),
```

Comment:
I think this should be changed to `ensure` or `enable` since `enabled` reflects the state of the system itself and can be determined by the `fips_enabled` fact.